packages/sdk/src/ordinals/Epoch.ts

```diff
@@ -0,0 +1,59 @@
+import { COIN_VALUE, SUBSIDY_HALVING_INTERVAL } from "./constants";
+// eslint-disable-next-line import/no-cycle
```

Comment:
is it possible to have a one-directional class flow? I'm still trying to understand why each class needs to return an instance of each other but don't have the full context yet

Reply:
Because for instance an `Epoch` will have a starting `Sat` and a `Sat` will belong to an `Epoch` 🤔